tools/autograd/derivatives.yaml

```diff
@@ -1181,6 +1181,12 @@
 - name: relu_(Tensor(a!) self) -> Tensor(a!)
   self: threshold_backward(grad, result, 0)
 
+- name: swish(Tensor self) -> Tensor
+  self: swish_backward(grad, self)
+
+- name: swish_(Tensor(a!) self) -> Tensor(a!)
```

Comment:
@albanD 
Will this work? For inplece, since the self here is not original input, but forward of input, it seems to me that this won't work.

Reply:
After discussion, this is not possible as we can't compute the derivative without the original value :'(  (the function is not bijective) cc @vadimkantorov 
So you can replace it with `not_implemented("swish_")`.
Not sure how to add a note here to mention that it is not possible to do so, maybe `not_implemented("swish_ (inplace version cannot compute the gradient, use out of place)")` will give a better error message.